stbt_control_relay.py

```diff
@@ -33,13 +33,13 @@
 from __future__ import absolute_import
 from builtins import *  # pylint:disable=redefined-builtin,unused-wildcard-import,wildcard-import,wrong-import-order
 import argparse
+import logging
 import os
 import re
 import signal
 import socket
 import sys
 
-import _stbt.logging
```

Comment:
Yes, this is sensible.  We can think about doing this more widely in the future.